src/graphql/queries/__tests__/GetArticle.js

```diff
@@ -0,0 +1,49 @@
+import gql from 'util/GraphQL';
+import { loadFixtures, unloadFixtures } from 'util/fixtures';
+import fixtures from '../__fixtures__/GetArticle';
+
+describe('GetCategory', () => {
```

Comment:
`GetArticle`~

Actually we have a "GetReplyAndGetArticle.js" test. I think we should move all `GetArticle` tests & fixtures inside that file to here as well.